crossbeam-deque/src/deque.rs

```diff
@@ -284,6 +284,28 @@ impl<T> Worker<T> {
         }
     }
 
+    /// Checks if the worker and the provided stealer are pointing to the same underlying queue.
```

Comment:
I feel it would be easier to understand documentation something like "returns true if the given stealer was derived from this worker".

Reply:
```suggestion
    /// Returns true if the given stealer was derived from this worker.
```